syntax = "proto3";
// Generated Webezy proto EDIT from CLI / Make manual edits here and run 'webezy build --manual'

import "protos/samplePkg.proto";





// [webezy] - Service description None
service SampleService {
	// [webezy] - SampleRPC
	rpc SampleRPC(domain.samplePkg.v1.SampleMsg) returns (domain.samplePkg.v1.SampleMsg);
	// [webezy] - GetSample
	rpc GetSample(domain.samplePkg.v1.SampleMsg) returns (domain.samplePkg.v1.OtherMsg);
}

